core/src/components/toggle/toggle.tsx

```diff
@@ -247,6 +248,14 @@ export class Toggle implements ComponentInterface {
   }
 
   private onClick = (ev: MouseEvent) => {
+    /**
+     * The haptics for the toggle is
+     * an iOS-only feature when tapped.
+     * As a result, it should be
+     * disabled on Android.
```

Comment:
```suggestion
     * The haptics for the toggle on tap is
     * an iOS-only feature. As such, it should
     * only trigger on iOS.
```

Just trying to clear up the grammar here a little bit to make the intention clearer.